File: examples/museum/oseg_generated/csharp/CreateSpecialEvent_default_example.cs
using System;
using System.Collections.Generic;
using System.IO;

using Org.OpenAPIMuseum.Api;
using Org.OpenAPIMuseum.Client;
using Org.OpenAPIMuseum.Model;

public class CreateSpecialEventDefaultExample
{
    public static void Main()
    {
        var config = new Configuration();

        var specialEvent = new SpecialEvent(
            name: "Mermaid Treasure Identification and Analysis",
            location: "Under the seaaa 🦀 🎶 🌊.",
            eventDescription: "Join us as we review and classify a rare collection of 20 thingamabobs, gadgets, gizmos, whoosits, and whatsits, kindly donated by Ariel.",
            price: 0,
            dates: new List<DateOnly>
            {
                DateOnly.Parse("2023-09-05"),
                DateOnly.Parse("2023-09-08")
            },
            eventId: null
        );

        try
        {
            var response = new EventsApi(config).CreateSpecialEvent(
                specialEvent: specialEvent
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Events#CreateSpecialEvent: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: examples/museum/oseg_generated/python/CreateSpecialEventDefaultExample.py
from datetime import date, datetime
from pprint import pprint

from openapimuseum_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration()

with ApiClient(configuration) as api_client:
    special_event = models.SpecialEvent(
        name="Mermaid Treasure Identification and Analysis",
        location="Under the seaaa 🦀 🎶 🌊.",
        event_description="Join us as we review and classify a rare collection of 20 thingamabobs, gadgets, gizmos, whoosits, and whatsits, kindly donated by Ariel.",
        price=0,
        dates=[
            date.fromisoformat("2023-09-05"),
            date.fromisoformat("2023-09-08"),
        ],
        event_id=None,
    )

    try:
        response = api.EventsApi(api_client).create_special_event(
            special_event=special_event,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling Events#create_special_event: %s\n" % e)
